de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/concept/types/structural/ABeanStructuralElementInstance.java

```diff
@@ -131,6 +179,27 @@ public void remove(IBeanStructuralElementInstance bsei) {
 		return bseiHelper.getAllSubBeanSeis(sei, beanSeiClazz);
 	}
 	
+	@Override
+	public List<ABeanStructuralElementInstance> getChildren() {
+		BeanStructuralElementInstanceHelper bseiHelper = new BeanStructuralElementInstanceHelper();
+		return new ArrayList<ABeanStructuralElementInstance>(bseiHelper.getAllSubBeanSeis(sei, ABeanStructuralElementInstance.class));
+	}
+	
+	@Override
+	@XmlElement
+	public void setChildren(List<ABeanStructuralElementInstance> newBeanSeis) {
+		EList<StructuralElementInstance> currentChildren = sei.getChildren();

```

Comment:
Is it neccessary to use EList, would List work as well?

Reply:
I will change it to List